src/openforms/js/components/admin/form_design/variables/VariablesTable.js

```diff
@@ -89,6 +89,7 @@ const VariableRow = ({index, variable}) => {
       <td>{getFormDefinitionName(variable.formDefinition)}</td>
       <td>{variable.prefillPlugin}</td>
       <td>{variable.prefillAttribute}</td>
+      <td>{variable.prefillIdentifierRole || 'main'}</td>
```

Comment:
I first thought: shouldn't 'main' be translated?
But I suspect the attribute always has a proper value, if I look at the proptypes of the component

Reply:
Mhmm yes it's true, maybe all of them should use the translatable label instead of the value :thinking:

Reply:
On second thoughts, it looks like this atm:
![Screenshot from 2023-07-11 10-45-42](https://github.com/open-formulieren/open-forms/assets/19154114/9df53898-2403-416a-b32e-d8e3a700e30d)
So I will just add the label for the identifier role in this PR.